rfcs/repo-management/shield-issue-expectations.md

```diff
@@ -0,0 +1,99 @@
+
+#### Problem
+
+There are issues logged in that lack the complete information needed to address them.
+
+#### Proposed solution
+
+We have templates in place for bugs, accessibility requests, feature requests, etc. There might be an area of opportunity to improve them but we could also be more strict about only addressing those issues that have all the information asked for in the templates.
+
+#### Problem
+
+The expectation on which issues should be fixed immediately vs can be delayed vs will probably not get fixed in the short or medium term is not well defined.
+
+#### Proposed solution
+
+We should agree and write down which issues should definitely be fixed while a person is on Shield and which ones are not necessarily going to be fixed during this time frame. Here is a proposed set of expectations:
+
+> _Issues that need to be fixed ASAP:_
+>
+> - Accessibility issues that affect our accessibility score card.
+> - Bugs identified as high impact or blocking.
+> - Bugs identified as regressions of previous functionality
+>
+> _Issues for which we encourage contributions that we are happy to help guide and review:_
+>
+> - Bugs that are identified as not being high impact.
+> - Feature requests that have been discussed about and agreed on a solution with team members.
+>
+> _Issues that we need to address but can't currently:_
+>
+> - Bugs that require a breaking change in terms of the API interface and require a major version release.
+>
+> _Issues that we make no commitments on ever fixing:_
+>
+> - Feature requests that have not been discussed with team members.
+
+#### Problem
+
+Users of FluentUI that are logging issues in our repository have a different set of expectations regarding issue resolution than the ones we have as part of our Shield process.
+
+#### Proposed solution
+
+We should clearly communicate with our customers how we are looking at issues and set clear expectations on them. This would ideally be in the way of a wiki entry in our repository detailing our Shield process with the expectations on issue resolution there.
+
+_TO-DO: Write up the wiki entry detailing the Shield process._
+
+## Pros and Cons
+
+The benefits of going with the proposed solutions are numerous:
+
+- There is a clearer, written-down set up expectations for both people on Shield and users logging issues regarding the incoming issues when someone is on Shield.
+- We are actively encouraging community contribution.
+- We are avoiding the duplication of efforts across PM and Engineering.
+
+This approach, however, carries some risks with it:
+
+- We have to be on top of community contributions if we start to encourage those, as sitting on a PR we encouraged will just make the author not want to contribute again.
```

Comment:
doing proper OSS is not an easy task and would need extensive time investment from our side. Do you have any actionables on top of your mind you'll like to propse/implement? 

- for example, having office hours with core team is a huge advantage of getting folks on board from my experience

Reply:
The office hours idea sounds interesting--do you have an example in mind of another open source library that has a good pattern for this?

Reply:
Office hours could be a good way to work through lower priority issues. When an issue gets sorted into the "lower priority" column we could follow up in the comments with our next scheduled office hours and encourage the reporter to attend to get started on the issue.

Some of our issues come from people external to Microsoft. Would office hours be open to these people?